c10/core/TensorImpl.h

```diff
@@ -552,12 +552,7 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
     return sizes_default();
   }
 
-  c10::SymIntArrayRef sym_sizes() const {
-    if (C10_UNLIKELY(
-            sizes_strides_policy_ >=
-            static_cast<uint8_t>(SizesStridesPolicy::CustomSizes))) {
-      return sym_sizes_custom();
-    }
+  virtual c10::SymIntArrayRef sym_sizes() const {
```

Comment:
Why do we want this to be virtual now, instead of doing the policy thing that we do with all of our other de-virtualized methods?

(I need to fix it up for functionalization, which will be pretty easy - just curious on the reasoning)

Reply:
And do we even need a `sym_sizes_custom()` anymore if `sym_sizes()` is virtual?

Reply:
@bdhirsh python tensor subclasses and LTC want to do different things when policy is set to `CustomSizes` and there's no easy way to implement it via `_custom` so we had to make `sym_sizes()` virtual for now